docs/quickstart.html

```diff
@@ -304,15 +304,24 @@
 </p>
 
 <pre>
-&gt; <b>cat /tmp/file-input.txt | ./bin/kafka-console-producer --broker-list localhost:9092 --topic streams-file-input</b>
+&gt; <b>bin/kafka-topics.sh --create \</b>
```

Comment:
I see. Originally we were relying on broker-side auto topic creation, but this maybe more clear.

BTW could you change line 302 a bit with double-quotes: `"streams-file-input"`, since `<b>..<\b>` seems not emphasize the name properly on the browser.